src/main/java/hudson/plugins/gradle/injection/MavenAbstractBuildRunListener.java

```diff
@@ -0,0 +1,18 @@
+package hudson.plugins.gradle.injection;
+
+import hudson.Extension;
+import hudson.maven.AbstractMavenBuild;
+import hudson.model.TaskListener;
+import hudson.model.listeners.RunListener;
+
+@Extension
+public class MavenAbstractBuildRunListener extends RunListener<AbstractMavenBuild<?, ?>> implements MavenInjectionAware {
+
+    @Override
+    public void onStarted(AbstractMavenBuild abstractMavenBuild, TaskListener listener) {
+        super.onInitialize(abstractMavenBuild);
```

Comment:
This is not needed, as we are not doing any decoration of the existing runner. All runners are invoked sequentially by Jenkins. `RunListener` is an abstract class with an empty implementation.

Reply:
I agree, that it's not needed, but I've found one or two implementations that do have it (and don't modify anything as well) and I thought keeping if it does something in future. 

Do you feel strongly about not having it here?

P.S. The invoked one should actually be `onStarted`, forgot to change it, but this is irrelevant if we remove it.

Reply:
Since this call doesn't do anything, I think it's a noise and we need to remove it.

Reply:
Refactored to one generic listener without this cal